src/main/java/gui/ClockController.java

```diff
@@ -89,8 +96,11 @@ private void switchCircle(GraphicsContext gc) {
 
     @FXML
     void startClock(MouseEvent event) {
+        PomodoroRunner pomodoroRunner = new PomodoroRunner();
```

Comment:
I think it would be easier to update the label in the AnimationTimer's loop, rather than opening another thread. `elapsedTime` reps the number of nanoseconds that has passed, you could pass it to the runner to format it.

Reply:
Sorry what's the runner?

Reply:
Ah I see, I'll try that out